client/src/components/SchemaOrg/OrganizationForm.vue

```diff
@@ -6,11 +6,14 @@
             <span v-b-tooltip.hover title="Hide Attribute"
                 ><FontAwesomeIcon icon="eye-slash" @click="onHide(attribute.key)"
             /></span>
+            <div v-if="currentErrors[attribute.key]" class="error">{{ currentErrors[attribute.key] }}</div>
             <b-form-input
                 :id="attribute.key"
                 v-model="currentValues[attribute.key]"
                 :placeholder="'Enter ' + attribute.placeholder + '.'"
-                :type="attribute.type">
+                :type="attribute.type"
+                :state="currentErrors[attribute.key] ? false : null"
+                @click="removeErrorMessage(attribute.key)">
```

Comment:
```suggestion
                @focus="removeErrorMessage(attribute.key)">
```

focus also works with keyboard navigation